athena/llm_core/llm_core/utils/llm_utils.py

```
@@ -1,14 +1,11 @@
-from typing import Type, TypeVar, List
+from typing import List, TypeVar
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)